internal/models/task_user.go

```diff
@@ -0,0 +1,11 @@
+package models
```

Comment:
You don't need to create a file for each model, have a models.go file and wrap all the models inside it.